versions.props

```diff
@@ -40,7 +40,7 @@ net.jcip:jcip-annotations = 1.0
 one.util:streamex = 0.7.3
 org.apache.commons:commons-pool2 = 2.9.0
 org.apache.commons:commons-lang3 = 3.12.0
-org.assertj:assertj-* = 3.20.1
+org.assertj:assertj-* = 3.20.2
```

Comment:
we may have to cut a `com.palantir.safe-logging` release with the assertj ABI revert as https://github.com/palantir/safe-logging/pull/542 bumped a bunch of folks to assertj 3.20.0

Reply:
yep:

```
java.lang.NoSuchMethodError: 'org.assertj.core.api.ThrowableAssert com.palantir.logsafe.testing.Assertions.assertThatThrownBy(org.assertj.core.api.ThrowableAssert$ThrowingCallable)'
	at com.palantir.logsafe.testing.Assertions.assertThatLoggableExceptionThrownBy(Assertions.java:58)
```